internal/pkg/plan/rename_plan_test.go

```diff
@@ -63,16 +63,19 @@ func TestRenameAllPlan(t *testing.T) {
 		actions: []*RenameAction{
 			{
 				OldPath:     "my-main-branch",
+				RenameFrom:  "my-main-branch",
 				NewPath:     "main",
 				Description: "my-main-branch -> main",
 			},
 			{
-				OldPath:     "main/extractor/keboola.ex-db-mysql/my-table",
+				OldPath:     "my-main-branch/extractor/keboola.ex-db-mysql/my-table",
+				RenameFrom:  "main/extractor/keboola.ex-db-mysql/my-table",
 				NewPath:     "main/extractor/keboola.ex-db-mysql/789-tables",
 				Description: "main/extractor/keboola.ex-db-mysql/{my-table -> 789-tables}",
 			},
 			{
-				OldPath:     "main/extractor/keboola.ex-db-mysql/789-tables/rows/my-row",
+				OldPath:     "my-main-branch/extractor/keboola.ex-db-mysql/my-table/rows/my-row",
+				RenameFrom:  "main/extractor/keboola.ex-db-mysql/789-tables/rows/my-row",
```

Comment:
Vysvetlim to na tomto teste:
- premenovava sa config row
- z `my-main-branch/extractor/keboola.ex-db-mysql/my-table/rows/my-row`
- na `main/extractor/keboola.ex-db-mysql/789-tables/rows/12-users`
- v samotnom config row sa premenovalo iba `my-row -> 12-users`
- ostatne zmeny pred tym su zmeny v ceste ku `parent` objektom -> tie sa vykonaju predtym.

Doteraz obsahovala `OldPath` staru cestu k objektu, ale uz po premenovani rodicov.

Upravul som to, aby `OldPath` obsahovala nezmenenu staru cestu.
A pridal som `RenameFrom`, kde je ta hodnota, co bola predtym v `OldPath`, ... tj. `OldPath` ale `parents` adresare su uz premenovane.

Potrebujem to pre dalsiu pracu, ... na update `shared_code_path`, ak sa shared code pemenoval.